ambry-tools/src/main/java/com.github.ambry/tools/perf/rest/NettyPerfClient.java

```diff
@@ -281,30 +303,39 @@ private NettyPerfClient(String host, int port, String path, int concurrency, Lon
       this.totalSize = 0;
       chunk = null;
     }
-    sslFactory = sslPropsFilePath != null ? new SSLFactory(
-        new SSLConfig(new VerifiableProperties(Utils.loadProps(sslPropsFilePath)))) : null;
+    if (sslPropsFilePath != null) {
+      SSLConfig sslConfig = new SSLConfig(new VerifiableProperties(Utils.loadProps(sslPropsFilePath)));
+      if (useOpenSsl) {
+        if (SslContext.defaultClientProvider() == SslProvider.JDK) {
+          throw new IllegalStateException("OpenSsl not supported. netty-tcnative might not be loaded correctly");
+        }
+        logger.info("Using this buffer pool: " + ByteBufAllocator.DEFAULT);
```

Comment:
use formatting

Reply:
I just removed these log messages. They were mostly for my debugging while i was writing this